x-pack/plugins/secops/public/components/timeline/body/index.tsx

```diff
@@ -151,39 +153,42 @@ export class Body extends React.PureComponent<Props, State> {
           width={columnWidths}
         >
           <EuiFlexGroup data-test-subj="events" direction="column" gutterSize="none">
-            {data.map(ecs => (
-              <EuiFlexItem data-test-subj="event" grow={true} key={ecs._id!}>
-                {getRowRenderer(ecs, rowRenderers).renderRow(
-                  ecs,
+            {data.map(ecsEdges => (
```

Comment:
I would prefer not to change the type here from `Ecs` to `EcsEdges`, because the latter feels like it's leaking transport-layer data types into the data contract(s).

Reply:
as we worked together, we introduced `memoizeOne` to avoid a new reference of our ECS data passed in `timeline/body` and like that we can keep `timeline/body` props the way they were.

Sorting, loadMore is still working with the memoization of the ECS data